pkg/ext-proc/handlers/request.go

```diff
@@ -67,14 +67,14 @@ func (s *Server) HandleRequestBody(reqCtx *RequestContext, req *extProcPb.Proces
 		requestBody, err = json.Marshal(rb)
 		if err != nil {
 			klog.V(logutil.DEFAULT).ErrorS(err, "Error marshaling request body")
-			return nil, fmt.Errorf("error marshaling request body: %v", err)
+			return nil, errutil.Error{Code: errutil.Internal, Msg: fmt.Sprintf("error marshaling request body: %v", err)}
```

Comment:
Why is this considered as Internal error instead of invalid request? Is it because we added new fields to rb?

Reply:
Resolved as this would indicate errors in EPP, which is an internal error.